lib/pages/withoutProps/redirects.js

```diff
@@ -1,10 +1,32 @@
+const getDefaultLocaleRedirectForI18n = require("../../helpers/getDefaultLocaleRedirectForI18n");
 const isDynamicRoute = require("../../helpers/isDynamicRoute");
 const pages = require("./pages");
 
-const redirects = [];
+let redirects = [];
+
+/** withoutProps pages
+ *
+ * Page params {
+const getDefaultLocaleRedirectForI18n = require("../../helpers/getDefaultLocaleRedirectForI18n");
+ *    route -> '/about', '/initial/[id]'
+ *    filePath -> 'pages/about.html', 'pages/initial[id].html'
+ * }
+ *
+ * Page params in i18n {
+ *    route -> '/en/about', '/fr/initial/[id]'
+ *    filePath -> 'pages/en/about.html', 'pages/fr/initial[id].html'
+ * }
+ **/
 
 pages.forEach(({ route, filePath }) => {
-  // Only create redirects for pages with dynamic routing
+  const target = filePath.replace(/pages/, "");
+
+  // Add redirect for naked route -> `${defaultLocale}/redirect` (if i18n)
+  redirects = redirects.concat(
+    getDefaultLocaleRedirectForI18n(route, null, target)
+  );
```

Comment:
this is interesting to me because originally i went the concat route after you made some comments about wanting to remove as many conditionals as possible. not sure on this one, will circle back

Reply:
now that i've circled back, i think this is [in some way] related to your other comment re: getStaticPropsWithRevalidate and moving logic from redirects.js to pages.js, and right now i'm leaning towards keeping logic in redirects.js right now with the potential of changing it in the future if we pivot to copying pages over instead

that said, if we want to get rid of the `concat`, i think there are a couple ways we can do that - both with and without a conditional